package.json

```diff
@@ -19,17 +19,20 @@
     "@types/react-dom": "^17.0.8",
     "@types/uuid": "^8.3.0",
     "eslint-config-prettier": "^8.3.0",
+    "jest": "26.6.0",
```

Comment:
you don't need `jest` right now.
testing framework already embedded in `react-scripts`